### DISCUSSION TITLE
SDR for audio source separation

### DISCUSSION THREAD

**Discussion by sophia1488** [asteroid-team/asteroid#600] [Q&A]
Hi, I'm doing audio source separation (speech + pop music), and my target is speech, so there's no permutation problem.
I hope to evaluate with SDR, SIR, and SAR.
I saw the example in [metrics.py](https://github.com/asteroid-team/asteroid/blob/master/asteroid/metrics.py), and I modified `mix`, `clean`, and `est` to be of the same shape.
I wonder if the result is reasonable.  If so, I could use it directly, which is great!
Or should I modify mir_eval function?

```py
from asteroid.metrics import get_metrics
mix = np.random.randn(2, 16000)        <--- 2: stereo (2-channel)
clean = np.random.randn(2, 16000)
est = np.random.randn(2, 16000)
metrics_dict = get_metrics(mix, clean, est, sample_rate=8000,
                           metrics_list='all')
pprint.pprint(metrics_dict)
```

Thank you!

**Comment by mpariente** [comment]
It'll compute the metrics for channels independently, right? I'm not sure this is what you want, is it?
